lib/Driver/ToolChains.cpp

```diff
@@ -166,6 +166,17 @@ void ToolChain::addCommonFrontendArgs(const OutputInfo &OI,
     arguments.push_back("-disable-objc-interop");
   }
 
+  // Add flags for C++ interop.
+  if (inputArgs.hasArg(options::OPT_enable_experimental_cxx_interop)) {
+    arguments.push_back("-enable-cxx-interop");
+  }
+  if (const Arg *arg =
+      inputArgs.getLastArg(options::OPT_experimental_cxx_stdlib)) {
```

Comment:
This seems oddly formatted, `clang-format` please?

Reply:
Done.

(The formatting that clang-format applied also seems a bit strange to me -- but who am I to argue with clang-format?)